app/views/plants/waterme.html.erb

```diff
@@ -0,0 +1 @@
+<h1>u mnie dziala</h1>
```

Comment:
Trzeba dodać nową linie po tym nagłówku :)

Reply:
`rubocop` by to wyłapał: polecam uruchamianie go przed wystawianiem pull requesta i przed mergowaniem :smile:

Reply:
ok poprawie

Reply:
Jednak nie bo to w widoku xD